gui/default/assets/lang/lang-en.json

```diff
@@ -208,6 +214,9 @@
     "Incoming Rate Limit (KiB/s)": "Incoming Rate Limit (KiB/s)",
     "Incorrect configuration may damage your folder contents and render Syncthing inoperable.": "Incorrect configuration may damage your folder contents and render Syncthing inoperable.",
     "Incorrect user name or password.": "Incorrect user name or password.",
+    "Ineligible WebAuthn Credentials": "Ineligible WebAuthn Credentials",
```

Comment:
Mention passkeys here as well?

Reply:
I considered it but felt it's a bit redundant because the heading "WebAuthn Credentials (Passkeys)" is right there just above this table. But ok, done in 23cd64bcbc179b4f8a3531d5c5b300cc008b0db5.